WordPressCom-Stats-iOS/StatsEphemory.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+@interface StatsEphemory : NSObject
```

Comment:
I like the name but for for future developers that need to maintain the code in the future, StatsCache will be more obvious no?